### PR TITLE
LPS-118606 Add field property to MultiBar chart cases

### PR DESCRIPTION
**Pull request by alinedoleron**
Related issue: https://issues.liferay.com/browse/LPS-118606

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/chart/bar/MultiBarChart.es.js [hunk 1]
```diff
@@ -96,18 +94,26 @@ export default ({data, field, height, structure, width}) => {
...
+						<li key={`item-${index}`}>
+							<svg height="12" width="12">
+								<rect
+									fill={entry.color}
+									height="12"
+									width="12"
+								/>
+							</svg>
+							<span>
+								{label && label.length > 44
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/chart/bar/MultiBarChart.es.js:110]: @alinedoleron, what `44` means? Please add it to a constant.

**Comment by alinedoleron** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by alinedoleron** [comment]
New pr: https://github.com/natocesarrego/liferay-portal/pull/754

**Pull request closed without merging**